include/libsodium/sodium/version.h

```diff
@@ -0,0 +1,6 @@
+/*
+ * \brief  Dummy version header needed by libsodium
+ * \author Stefan Kalkowski
```

Comment:
@skalk you, here? ;-)

Reply:
I think I ported libsodium initially, then @skalk cleaned it up, and I cherry-picked his commit. I reset the author on his commit because I disabled some of the global initialization code in libsodium but I didn't want someone else to take the blame if things break.

Reply:
Thanks for the clarification. I merged both commits to master now.